contracts/Gauge.sol

```diff
@@ -537,13 +542,13 @@ contract Gauge is BaseGauge, IGauge {
     /**
      * @notice
      * set reward manager
-     * @dev Can be called by rewardManager or gov
+     * @dev Can be called by rewardManager or owner
```

Comment:
Suggestion for [this line](https://github.com/yearn/veYFI/pull/99/files#diff-64386ca189f0bc28080b2cf6cf03e6a9149cc593ea9ed29c10aa4ac80e394205R225) or similar.

I would always use `{...}` in `if/else/for/while/etc`. Sometime I had an issue with a linter/formatter that formatted an `if` without `{..}` incorrectly.

Reply:
Seems this comment didn't went to the right line